docs/theming/developers.md

```diff
@@ -35,8 +38,26 @@ hasToc: true
 
 ## Using the theming system
 
-### Themable containers
+### Themeable containers
+
+A document, parent HTML element, or special provider elements such as `<rh-surface>` can all be considered a themeable containers.  A themeable container is an element that can have custom classes attached which provide values for the relevant CSS color properties in a [theme](themes).
```

Comment:
this part isn't true, at least not yet. when style queries reach GA, we'll be able to say this, but only under the condition that the developer applies our special context css custom props.

Reply:
see 70da4010